### DISCUSSION TITLE
Is it possible to expose the Document Information (XMP) just like with the Outline?

### DISCUSSION THREAD

**Discussion by leonsavvidi** [espresso3389/pdfrx#177] [Ideas]
Even a raw data (decrypted) would suffice

**Comment by espresso3389** [comment]
An interesting fact is that pdfium does not provide API to get XMP from PDF file :(
So we should use [dart_pdf_reader](https://pub.dev/packages/dart_pdf_reader) or such to load XMP manually...

**Reply by espresso3389** [reply]
The flow may be like:
- Parse PDF stream to get document
- Get catalog
- Get "Metadata" from catalog's dictionary
- Get metadata's decoded/decrypted stream

Anyway, I didn't test it by myself...
